content/sensu-core/1.8/migration.md

```diff
@@ -0,0 +1,380 @@
+
+{{< highlight shell >}}
+# Requires curl and jq
+curl -s http://127.0.0.1:4567/settings | jq . > sensu_config_original.json
+{{< /highlight >}}
+
+Now install and run the translator.
+
+{{< highlight shell >}}
+# Install dependencies
+yum install -q -y rubygems ruby-devel
+
+# Install sensu-translator
+gem install sensu-translator
+
+# Translate the config in /etc/sensu/conf.d to Sensu Go, output to /sensu_config_translated
+sensu-translator -d /etc/sensu/conf.d -o /sensu_config_translated
+{{< /highlight >}}
+
+If translation is successful, you should see a few callouts followed by `DONE!`.
+
+{{< highlight shell >}}
+Sensu 1.x filter translation is not yet supported
+Unable to translate Sensu 1.x filter: only_production {:attributes=>{:check=>{:environment=>"production"}}}
+DONE!
+{{< /highlight >}}
+
+Combine your config into a sensuctl-readable format.
+Note that, for use with `sensuctl create`, Sensu Go resource definitions in JSON format should _not_ have a comma between resource objects.
+
+{{< highlight shell >}}
+find sensu_config_translated/ -name '*.json' -exec cat {} \; > sensu_config_translated_singlefile.json
+{{< /highlight >}}
+
+While most attributes are ready to use as-is, you'll need to adjust your Sensu Go configuration manually to migrate some of Sensu's features.
+
+### 2. Translate checks
+
+Review your Sensu 1.x check configuration for the following attribute, and make the corresponding updates to your Sensu Go configuration.
+
+| 1.x attribute | Manual updates required in Sensu Go config |
+| ------------- | ------------- |
+`::: foo :::` | Update the syntax for token substitution from triple colons to using curly braces. For example: `{{{ foo }}}`
+`stdin: true`  | No updates required. Sensu Go checks accept data on stdin by default.
+`handlers: default` | Sensu Go no longer has the concept of a default handler, so you'll need to create a handler called default to continue using this pattern.
+`subdues` | Check subdues are not available in Sensu Go.
+`standalone: true` | Standalone checks are no longer supported in Sensu Go, although [similar functionality can be achieved using role-based access control, assets, and entity subscriptions][26]. The translator assigns all 1.x standalone checks to a `standalone` subscription in Sensu Go, so you can add the `standalone` subscription to a Sensu Go agent configuration.
+`metrics: true` | See the section on [translating metric checks](#translating-metric-checks).
+`proxy_requests` | See the section on [translating proxy requests](#translating-proxy-requests-and-proxy-entities).
+`subscribers: roundrobin...` | Remove `roundrobin` from the subscription name, and add the `round_robin` check attribute set to `true`.
+`aggregate` | Check aggregates are supported through the [license-activated][40] [Sensu Go Aggregate Check Plugin][28].
+`hooks` | See the section on [translating hooks](#translating-hooks).
+`dependencies`| Check dependencies are not available in Sensu Go.
+

```

Comment:
Note on check token substitution to highlight.. maybe as a pro tip.  To access labels or annotations that include a "dot"  for ex: "sensio.io.json_attributes"   you need to use the built in go template "index" function 
ex: {{index .annotations "web_url"}}   would substitute the value of the "web_url" annotation
ex: {{index .annotations "production.ID"}}   would substitute the value of the "production.ID" annotation

Reply:
Added.